.github/workflows/pipeline.yml

```diff
@@ -415,42 +360,19 @@ jobs:
           script: |
             mkdir -p ~/www/${{ inputs.deploy_host }}/releases/${{ steps.commit.outputs.short }}
 
-      - name: Copy deployment for Next.js
-        if: ${{ env.framework == 'next' && env.ssr == 'true' }}
+      - name: Copy deployment
         uses: burnett01/rsync-deployments@5.2.1
         with:
           switches: -avzr
-          path: .next/standalone/
+          path: ${{ env.dist_path }}
```

Comment:
This should probably be set for each technology inside the pipeline to avoid breaking change in each project.
This way we need to add `dist_path` variable to each project before merging to master.

Reply:
I don't think we need anything for backwards compatibility because we're already setting defaults in the detect env action.